### PR TITLE
Add ambassador and maintainer roles

### PR DESCRIPTION
**Pull request by bstarling**

### CODE REVIEW

File: project-ambassador.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#### Opportunities:
+* Build deeper relationships with project team.
+* Participate in discussions to determine project direction and focus.
+* Hone your communication, collaboration and development skills.
+* Prepare for a role as a project lead.
+
+#### Responsibilities:
+* Able to commit at least 2-3 hours a week (on average) to the project.
+* Help new people get up and running
+* Point people to GitHub, find out what their strengths/interests are, how they’d like to be involved. Connect with people with tasks.
```
Comment (geckya) [inline project-ambassador.md]: Connect people with tasks.

File: project-ambassador.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Project Ambassador
+
+Ambassadors are members who support our project leads and maintainers to make sure the project is welcoming and well documented. They are on the front line helping new members determine if a specific project is a good fit for them. This is a great position for people who want to take on an expanded role but still new enough they are able to identify roadblocks for newcomers that others may not notice.
```
Comment (geckya) [inline project-ambassador.md]: ...but are still new enough...

File: project-ambassador.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+* Build deeper relationships with project team.
+* Participate in discussions to determine project direction and focus.
+* Hone your communication, collaboration and development skills.
+* Prepare for a role as a project lead.
+
+#### Responsibilities:
+* Able to commit at least 2-3 hours a week (on average) to the project.
+* Help new people get up and running
+* Point people to GitHub, find out what their strengths/interests are, how they’d like to be involved. Connect with people with tasks.
+* Advocate for the project in the larger community
```
Comment (geckya) [inline project-ambassador.md]: missing "."

File: project-ambassador.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+* Listed in the project readme as a resource to help new members through github and/or submitting their first pull request, getting access to repos, servers etc.
```
Comment (geckya) [inline project-ambassador.md]: GitHub

File: project-ambassador.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+* This is not necessarily a technical role. You do not have to have all the answers but should you should be willing to go the extra mile to make warm introductions and help people find the right resource when necessary.
+* Identify individuals whose work should be elevated on the (as-of-yet-non-existent) blog, and on Partially Derivative
```
Comment (geckya) [inline project-ambassador.md]: "."

File: project-ambassador.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+* Be patient, kind and welcoming.
+
+#### First tasks:
+* Work with project lead to make the announcement in Slack, let people know they can reach out to you.
+* Add yourself to the project readme.
+* Start engaging with newcomers
```
Comment (geckya) [inline project-ambassador.md]: .

File: project-maintainer.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+## Project Maintainer
+
+Maintainers are members who support our project leads to review all code being submitted to our repositories. Maintainers may work across projects or specialize on a single project. They are available in slack to answer questions and help people with pull requests.
```
Comment (geckya) [inline project-maintainer.md]: Slack

File: project-maintainer.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+Guiding philosophy taken from our readme:
+* **We believe good code is reviewed code.** All commits to this repository are approved by project maintainers and/or leads. The goal here is *not* to criticize or judge your abilities! Rather, sharing insights and achievements. Code reviews help us continually refine the project's scope and direction, as well as encourage the discussion we need for it to thrive.
```
Comment (geckya) [inline project-maintainer.md]: Rather, we aim to share insights and achievements.

File: project-maintainer.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Opportunities:
+* Build deeper relationships with project team.
+* Participate in discussions to determine project direction and focus.
+* Hone your communication, collaboration and development skills.
+* Reviewing code can be much more challenging than writing code. Grow your skillet by teaching and mentoring.
```
Comment (geckya) [inline project-maintainer.md]: skill set

File: project-maintainer.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+#### Responsibilities:
+* Able to commit at least 5 hours a week (on average) to the project.
+* Listed in the project readme as a resource.
+* Perform code reviews, manage pull requests & branches.
+* Make sure issue list stays organized. Close old issues, add proper issue labels, assign issues.
+* Open `beginner-friendly` and `first-pir`. Mentor people who volunteer to take on these issues.
```
Comment (geckya) [inline project-maintainer.md]: first-pr

File: project-maintainer.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+* Define milestones and help people to meet them.
+* Be patient, kind and welcoming.
+
+#### First tasks:
+* Work with project lead to make the announcement in Slack, let people know they can reach out to you.
+* Add yourself to the project readme.
+* Start reviewing pull requests & grooming issues
```
Comment (geckya) [inline project-maintainer.md]: .

**Comment by bstarling** [comment]
Updated!

**Pull request merged**